package.json

```diff
@@ -27,6 +27,9 @@
     ".": {
       "sass": "./_index.scss"
     },
+    "scss/*": {
```

Comment:
```suggestion
    "./scss/*": {
```

Reply:
Good catch

I would keep the scss exports before the js for clarity